src/qtiCommonRenderer/tpl/img.tpl

```diff
@@ -8,5 +8,9 @@
     {{#if attributes.height}}height="{{attributes.height}}" {{/if}}
     {{#if attributes.width}}width="{{attributes.width}}" {{/if}}
```

Comment:
Shall we remove these two then maybe?

Reply:
I prefer to save this, because in a case if in `css` we will have some styles for images this styles will apply and rewrite styles from attributes
![image](https://user-images.githubusercontent.com/25976342/81283906-13b64e00-9066-11ea-9bc7-97055ba98259.png)

Reply:
In that case, isn't it better to keep only the attributes, and remove the `style` whatsover?